packages/noports_core/lib/src/sshnp/sshnp_impl.dart

```diff
@@ -246,44 +241,7 @@ abstract class SSHNPImpl implements SSHNP {
         .toLowerCase();
     logger.info('Received $notificationKey notification');
 
-    bool connected = false;
-
-    if (notification.value == 'connected') {
-      connected = true;
-    } else if (notification.value?.startsWith('{') ?? false) {
-      late final Map envelope;
```

Comment:
This payload code was leaking implementation details, so it was replaced with the abstract handleSshnpdPayload which is implemented by the Payload Handler mixins